### PR TITLE
email notifications on privileged actions configuration T7197

### PR DESCRIPTION
**Pull request by Reception123**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -3684,7 +3685,9 @@
 	
 	// Rights
 	'+wgAvailableRights' => [
-		'default' => [],
+		'default' => [
+			'watched-person',
```
Comment (RhinosF1) [inline LocalSettings.php:3689]: Others then need moving to + codes. I wonder if there's a way to just add straight to the global group.

File: LocalSettings.php [hunk 2]
```diff
@@ -4489,6 +4492,14 @@
 	],
 ];
 
+// Email notifications on privileged actions configuration
+'wgMirahezeMagicLogEmailConditions' => [
+	'default' => [
+		'right' => 'watched-person',
```
Comment (supertassu) [inline LocalSettings.php:4498]: wgMirahezeMagicLogEmailConditions supports multiple watched groups, those values need to be wrapped inside an array

File: LocalSettings.php (hunk 1 above)
Comment (dmehus) [reply LocalSettings.php:3689]: If it's in `$wgAvailableRights` for `default`, it should be possible to then just add that user right in Special:GlobalGroupPermissions, I believe

File: LocalSettings.php [hunk 3]
```diff
@@ -4487,6 +4491,16 @@
 			'XMP' => false,
 		],
 	],
+
+// Email notifications on privileged actions configuration
```
Comment (Universal-Omega) [inline LocalSettings.php:4495]: Indentation is off I think? Needs one additional tab on everything it seems. And removal of tab on a bracket.

Comment (Reception123) [reply LocalSettings.php:4495]: Thanks, will fix that when I change my email to Owen's after testing.

**Pull request merged**